deploy-partner-pods.sh

```diff
@@ -0,0 +1,40 @@
+mkdir -p ./temp
+
+if [[ -z "${NAMESPACE_TO_GENERATE}" ]]; then
+    export NAMESPACE_TO_GENERATE="tnf"
+fi
+
+export res=$(oc get namespace $NAMESPACE_TO_GENERATE 2>&1)
+if [[ ${res#Error from server (NotFound)} != ${res} ]] || [[ ${res#No resources found} != ${res} ]]; then
+    cat ./local-test-infra/namespace.yaml | ./script/mo > ./temp/rendered-namespace.yaml
+    oc apply -f ./temp/rendered-namespace.yaml
+    rm ./temp/rendered-namespace.yaml
+else
+    echo "namespace ${NAMESPACE_TO_GENERATE} already exists, no reason to recreate"
+fi
+
+if [[ -n "${TNF_MINIKUBE_ONLY}" ]] && [[ ${TNF_MINIKUBE_ONLY} = "true" ]]; then
+    echo "Running with Minikube, no FsDiff pod created"
+else
+    export res=$(oc get deployment -n $NAMESPACE_TO_GENERATE node-master 2>&1)
+    if [[ ${res#Error from server (NotFound)} != ${res} ]] || [[ ${res#No resources found} != ${res} ]]; then
+        cat ./local-test-infra/fsdiff-deployment.yaml | ./script/mo > ./temp/rendered-fsdiff-template.yaml
+        oc apply -f ./temp/rendered-fsdiff-template.yaml
+        rm ./temp/rendered-fsdiff-template.yaml
+    else
```

Comment:
Thinking more on this, it seems to be that skipping is probably not what we want for the container case at least. The container may contain a different versions of partner pods. Common use case would be clean the deployments/pods already there in the namespace and redeploy with the version from the container image. @isaacdorfman @edcdavid @HamadiSE @wying3 Comments?

Reply:
I did this